src/java.base/share/classes/java/io/InputStream.java

```diff
@@ -55,6 +56,19 @@ public abstract class InputStream implements Closeable {
 
     private static final int DEFAULT_BUFFER_SIZE = 8192;
 
+    private SoftReference<byte[]> skipBufferReference;
+
+    private byte[] skipBufferReference(long remaining) {
+        int size = (int) Math.min(MAX_SKIP_BUFFER_SIZE, remaining);
```

Comment:
This is now redundant. Can change parameter to (int size) directly

Reply:
@liach done.